build/Makefile

```
@@ -856,3 +856,10 @@ test-gen-crd-client:
 		echo "$$diff_output"; \
 	fi
 
+update-agones: $(ensure-build-image)
```

```suggestion
install-release: $(ensure-build-image)
```
Maybe, as a better name?

---

modified here and other locations as well